Cargo.toml

```diff
@@ -7,8 +7,11 @@ keywords = ["TiKV"]
 name = "kvproto"
 
 [dependencies]
-# TODO: support 1.6
-protobuf = "~1.4"
+protobuf = "~2.0"
 futures = "0.1"
-grpcio = "0.2"
-raft = "0.2"
+grpcio = "0.3"
+
+# TODO: reset it to released version.
+[dependencies.raft]
+git = "https://github.com/pingcap/raft-rs"
+rev = "f271cd0dae4c868b2d92e19fe6c14e000598ea76"
```

Comment:
@BusyJay 

Can we release a new version for raft?

Reply:
Yes. It will be released after the docs related pull requests are merged.

Reply:
https://github.com/pingcap/raft-rs/pull/69 if you feel like giving a LGTM. :)

Reply:
It is merged now. :)